out-of-sync submodule branch for teaclave sgx sdk for v1.6.1

The source code for the latest mainnet release tagged v1.6.1 has the following `.gitmodules`: https://github.com/scrtlabs/SecretNetwork/blob/3cfc3495d7432a65b466906d7f8fea9a0c386444/.gitmodules#L1-L4

Building `libgo_cosmwasm.so` will fail with the following error:

```console
   Compiling lazy_static v1.4.0         
   Compiling base64 v0.12.3
error[E0432]: unresolved import `core::error`
  --> /go/src/github.com/enigmampc/SecretNetwork/third_party/incubator-teaclave-sgx-sdk/sgx_types/src/error.rs:19:11
   |
19 | use core::error::Error;
   |           ^^^^^ could not find `error` in `core`

   Compiling parity-wasm v0.41.0
For more information about this error, try `rustc --explain E0432`.
error: could not compile `sgx_types` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
```

Changing the branch from `master` to `secret-new-1.1.5` resolves the issue, like it has been done in more recent versions of the codebase.

I could make a pull request if needed.

--- a/.gitmodules
+++ b/.gitmodules
@@ -1,4 +1,4 @@
 [submodule "third_party/incubator-teaclave-sgx-sdk"]
 	path = third_party/incubator-teaclave-sgx-sdk
 	url = https://github.com/scrtlabs/incubator-teaclave-sgx-sdk
-	branch = master
+	branch = secret-new-1.1.5

